pkg/audit/manager.go

```diff
@@ -523,13 +529,40 @@ func (am *Manager) reviewObjects(ctx context.Context, kind string, folderCount i
 
 			results := ToResults(&augmentedObj.Object, resp)
 
-			if len(resp.Results()) > 0 {
-				err = am.addAuditResponsesToUpdateLists(updateLists, results, totalViolationsPerConstraint, totalViolationsPerEnforcementAction, timestamp)
+			// Expand object and review any resultant resources
+			allResps := []*rtypes.Responses{resp}
+			base := &mutationtypes.Mutable{
+				Object:    objFile,
+				Namespace: &ns,
+				Username:  "",
+				Source:    mutationtypes.SourceTypeGenerated,
```

Comment:
Should this be `Original`?

Reply:
Do we want to apply mutators to the generator before we expand it? The current code does not do this, as I assumed it would have been mutated by mutation webhook upstream, although this would not be the case for the CLI now that I think about it.

If we do want to mutate the generator first then, yes, this should be Original. If not, then I'll just take this line out as it doesn't actually doesn't matter since the generator isn't subject to mutation.

Reply:
Per our offline discussion, the CLI has been changed to mutate generators before expanding them.